dashboard/app/helpers/aichat_sagemaker_helper.rb

```diff
@@ -18,7 +18,7 @@ def self.create_sagemaker_client
   # Mistral-7B-Instruction LLM instruction format doc at https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.1.
   def self.format_inputs_for_sagemaker_request(aichat_params, stored_messages, new_message)
     all_messages = [*stored_messages, {role: USER, content: new_message}]
-    inputs = aichat_params[:systemPrompt]
+    inputs = aichat_params[:systemPrompt] + " "
```

Comment:
@fisher-alice unrelated change, but I noticed these were getting appended together without a space, does this seem ok?

Reply:
Sure. Thanks!